pkg/node/node.go

```diff
@@ -378,6 +386,32 @@ func unstageVolume(ctx context.Context, stagingPath, id string, logFields log.Fi
 	return device, nil
 }
 
+func removeRemnantMounts(ctx context.Context, stagingPath string, fs fs.Interface, logFields log.Fields) (string, error) {
+	log.WithFields(logFields).Infof("getting remnant mount")
+	mounts, found, err := getRemnantTargetMounts(ctx, stagingPath, fs)
+	if !found || err != nil {
+		return "", fmt.Errorf("could not reliably determine remnant mounts for path %s: %s", stagingPath, err.Error())
+	}
+
+	log.WithFields(logFields).Infof("%d remnant mount exist", len(mounts))
+	for _, mount := range mounts {
+		delete(logFields, "StagingPath")
```

Comment:
In the function  definition seeing the argument as  [stagingPath](https://github.com/dell/csi-powerstore/pull/190/files#diff-515b2d0691202c4ea5e578ec896258ae9c440c381b608ed1eb7116a27847057dR389)
And In the delete function, we are passing `stagingPath` as a string is both are the same ?

Reply:
The two are different. We remove "StagingPath" from logFields in this line, and add it back in line 408.

We remove StagingPath and replace it with RemnantPath, since this block deals with RemnantPath. After the operation is complete, RemnantPath is removed and StagingPath is added back.